psycop/common/feature_generation/sequences/test_patient_loaders.py

```diff
@@ -4,17 +4,17 @@
 from psycop.common.test_utils.str_to_df import str_to_pl_df
 
 
-def test_diagnosis_formatting():
+def test_diagnosis_preproessing():
     df = str_to_pl_df(
         """dw_ek_borger,datotid_slut,diagnosegruppestreng
     1,2023-01-01,A:DF431                    # Keep all, even though doesn't end with a hashtag
```

Comment:
We're not testing that this row is being kept anymore, because DF431 is present in both rows. Perhaps change it to another diagnosis?

Our tests also don't cover the new filtering behaviour.